packages/head/helpers/is-absolute-url.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Given a string,
+ * return true if the string is an absolute URL,
+ * or false otherwise.
+ *
+ * Uses regex so may not be 100% accurate.
+ * Based on https://github.com/sindresorhus/is-absolute-url
+ *
+ * @param string The URL to test
+ * @returns true if the URL is absolute, false otherwise
+ */
+function isAbsoluteUrl(string: string): boolean {
```

Comment:
Pulled in from hashicorp/dev-portal (and elsewhere).

Reply:
(Question 🙋) Can some examples of what is and isn't an absolute URL be added to the block comment above this function? That will help make it clear at a glance how this function works. I end up having to look it up every time. 😅

(Future 📌) Tests would be nice as well, but maybe this is a helper we should prioritize moving to [`@hashicorp/platform-util`](https://github.com/hashicorp/web-platform-packages/tree/main/packages/util) first? Sounds like it's very widely used!

Reply:
These examples look great! Very clear and super helpful. Thank you for adding 💖